src/main.lua

```diff
@@ -22,10 +22,11 @@ Vaults = Vaults or {}
 GatewayRegistry = GatewayRegistry or {}
 NameRegistry = NameRegistry or {}
 Epochs = Epochs or {}
+
+-- last known timestamps used to validate behavior to perform on new messages
+LastCreatedEpochIndex = LastCreatedEpochIndex or -1 -- TODO: consider making epochs 1 indexed and start at 0
```

Comment:
Not too much more time to decide on the TODO. If we make an adjustment here it likely would be disruptive in testnet but could work fine with a fresh slate in mainnet.

Reply:
agree - i think testnet would just "jump an epoch", but will ponder it more and put it for a separate PR

Reply:
i made the move to 1 based indexing - updates in https://github.com/ar-io/ar-io-network-process/pull/353/commits/11b9e1c33bb0975520a93ec5193b074670982a3f

I believe for testnet this should just "skip an epoch", we can validate in devnet